Tests/SwiftDocCTests/Rendering/RenderNodeTranslatorTests.swift

```diff
@@ -1057,5 +1057,15 @@ class RenderNodeTranslatorTests: XCTestCase {
         XCTAssertEqual(row.columns.first?.content.count, 1)
         XCTAssertEqual(row.columns.last?.size, 5)
         XCTAssertEqual(row.columns.last?.content.count, 3)
+        
+        guard case let .small(small) = discussion.content.last else {
```

Comment:
Should we not have a separate test from row and column one for this? I find that having smaller test cases helps pin out failures faster.

Reply:
Ah good catch- I meant to create a new one here. Should be fixed now. Thanks!